### ISSUE TITLE
RPM package warning: `legacy directory /var/run`

### ISSUE THREAD

**Issue by AlexRuiz7** [wazuh/wazuh-indexer#105]
## Description

Related issue: #99 

During RPM packages installation and removal, these warning show up:

`.rpm` package install:
```
[root@alma8 ~]# rpm -i wazuh-indexer-4.9.0-1.x86_64.rpm 
warning: %post(wazuh-indexer-4.9.0-1.x86_64) scriptlet failed, exit status 255
[/usr/lib/tmpfiles.d/wazuh-indexer.conf:1] Line references path below legacy directory /var/run/, updating /var/run/wazuh-indexer → /run/wazuh-indexer; please update the tmpfiles.d/ drop-in file accordingly.
```

`.rpm` package uninstall:
```
warning: file /var/run/wazuh-indexer: remove failed: No such file or directory
```

## Tasks

- Fix warnings.

**Comment by f-galland** [comment]
We need to update `/var/run` references to `/run` in all `rpm` package related files:

```shell
distribution/packages/src/common/systemd/wazuh-indexer.conf
distribution/packages/src/common/systemd/wazuh-indexer.service
distribution/packages/src/common/env/wazuh-indexer
distribution/packages/src/common/scripts/postrm
distribution/packages/src/rpm/init.d/wazuh-indexer
```

**Comment by f-galland** [comment]
Since the above files mostly pertain to the common build infrastructure, we might need to modify `deb` related files as well, including:
```shell
distribution/packages/src/deb/debmake_install.sh
```

**Comment by f-galland** [comment]
PR #119 fixes the `/var/run` related warnings, but we still observe the following when installing:

```shell
Running transaction
  Preparing        :                                                                                                                                                              1/1 
  Running scriptlet: wazuh-indexer-4.9.0-1.x86_64                                                                                                                                 1/1 
  Installing       : wazuh-indexer-4.9.0-1.x86_64                                                                                                                                 1/1 
  Running scriptlet: wazuh-indexer-4.9.0-1.x86_64                                                                                                                                 1/1 
warning: %post(wazuh-indexer-4.9.0-1.x86_64) scriptlet failed, exit status 255

Error in POSTIN scriptlet in rpm package wazuh-indexer
  Verifying        : wazuh-indexer-4.9.0-1.x86_64                                                                                                                                 1/1 

Installed:
  wazuh-indexer-4.9.0-1.x86_64                                                                                                                                                        

Complete!
```

**Comment by AlexRuiz7** [comment]
Is `coreutils` installed?

- https://documentation.wazuh.com/current/installation-guide/wazuh-indexer/step-by-step.html#nodes-installation
- https://access.redhat.com/solutions/5749321

**Comment by f-galland** [comment]
It seems exit code `255` is originated in a `command not found` error:
https://www.baeldung.com/linux/status-codes

**Comment by AlexRuiz7** [comment]
> It seems exit code `255` is originated in a `command not found` error: https://www.baeldung.com/linux/status-codes

Fixed in 3b4a1f5d45620879ef8840ce009560778c3d29cc
